rollup.config.js

```diff
@@ -65,16 +66,32 @@ const external = (id) =>
   !id.startsWith('\0') && !id.startsWith('.') && !id.startsWith('/');
 
 // Pragmatically create a Rollup config for each package
-const getRollupConfig = ({ pwd, babelConfigFile }) => {
+const getRollupConfig = ({ pwd, babelConfigFile, isModule }) => {
   const SOURCE_DIR = path.resolve(pwd);
+  const PRIVATE_PATH = `${SOURCE_DIR}/src/private.js`;
+
   // Get the package.json file
   const pkgConfig = require(`${SOURCE_DIR}/package.json`);
+
   // Relative input location for Rollup to bundle from
-  const input = [`${SOURCE_DIR}/src/index.js`, `${SOURCE_DIR}/src/private.js`];
+  const input = [`${SOURCE_DIR}/src/index.js`];
+
+  // Not all packages have private exports
+  if (fs.existsSync(PRIVATE_PATH)) {
+    input.push(PRIVATE_PATH);
+  }
 
-  // Shared Rollup plugins
+  // Shared Rollup plugins for code-only packages
```

Comment:
```suggestion
  // Shared Rollup plugins
```
This should go back to the previous comment. This change only has to do with detecting if there is a private input/output.